test/config/gatewayclass.yaml

```diff
@@ -86,19 +86,19 @@ spec:
 kind: GatewayClass
 apiVersion: gateway.networking.k8s.io/v1
 metadata:
-  name: upgrade
+  name: package
```

Comment:
shouldnt this be `lifecyle` as well ?

Reply:
> shouldnt this be `lifecyle` as well ?

This worries me a bit: I think `lifecycle` doesn't seem to have good prominence.
On the contrary I would think that `package lifecycle` might be better?